mne/channels/layout.py

```diff
@@ -445,10 +445,10 @@ def find_layout(info, ch_type=None, exclude='bads'):
         layout_name = 'magnesWH3600'
     elif has_CTF_grad:
         layout_name = 'CTF-275'
-    elif n_kit_grads <= 157:
-        layout_name = 'KIT-157'
-    elif n_kit_grads > 157:
-        layout_name = 'KIT-AD'
+    elif n_kit_grads > 0:
+        layout_name = _find_kit_system(info, n_kit_grads)
+        if layout_name == 'KIT-UMD':
+            return _auto_layout(info)
```

Comment:
There is some, but limited symmetry... here is a `raw.plot_sensors()` plot and a layout generated with MNE `_auto_topomap_coords()`:

![sensors](https://cloud.githubusercontent.com/assets/145771/16545684/ff03c456-40ff-11e6-8f2d-31dacbc2ecff.png)

![auto](https://cloud.githubusercontent.com/assets/145771/16545683/fba2f386-40ff-11e6-8a83-17ac3f14b8ed.png)